app/src/sharedTest/java/org/oppia/app/ongoingtopiclist/OngoingTopicListActivityTest.kt

```diff
@@ -44,7 +46,8 @@ import javax.inject.Singleton
 class OngoingTopicListActivityTest {
 
   private val internalProfileId = 0
-  @Inject lateinit var storyProfileTestHelper: StoryProgressTestHelper
+  @Inject
+  lateinit var storyProfileTestHelper: StoryProgressTestHelper
```

Comment:
@virajpatel2411 As discussed earlier, we were facing some issues with CircleCI, so for you you can make this change and push the code to this branch.

This should be in single line, something line this:
`@Inject lateinit var storyProfileTestHelper: StoryProgressTestHelper`

Reply:
@rt4914 Done the required change and committed.